### DISCUSSION TITLE
Vault 2023

### DISCUSSION THREAD

**Discussion by CarolusIII** [bramses/bramses-highly-opinionated-vault-2023#11] [General]
Hello Bramses, your detailed Readme of the Bramses' Highly Opinionated Vault 2023 got my attention.
I would love to learn more on your thoughts and ideas.
It is probably due to me being not that experienced in the use of github, but I really could not find any way to download a zipped version of your vault that I could use.
Hopefully you can help, but anyway I wish you a lot of success with your adventures.
I subscribed to your newsletter.
All the best,
Wim Claessen
Netherlands.

**Comment by CarolusIII** [comment]
Actually I found the reason why I could not find the green download button to your Obsidian 2023 vault.
I was my "Brave" Browser that did that.
Using the normal Chrome browser I got access right away.
Sorry for the confusion and thanks for your great work.
Happy now :)

**Comment by bramses** [comment]
@CarolusIII glad to hear it worked!
